README.md

```diff
@@ -119,6 +119,11 @@ Different flags (for the same filter target) combine with AND semantics. For
 example, `-metric-allowlist 'bytes_total$' -metric-blocklist imgopto` would only
 export metrics whose names ended in bytes_total, but didn't include imgopto.
 
+### Aggregate Datacenter
+
+Datacenter metrics can be aggregated to reduce the output size of the metrics
+endpoint by using the `-aggregate-datacenter` flag.
+
```

Comment:
I think the naming is especially difficult because it has already been termed `Aggregated`.
https://github.com/fastly/fastly-exporter/blob/f6a1c512da5ae7530d351fadcfd0b00e766f1c9f/pkg/realtime/response.go#L9-L13

The way I understand this is we two choices for metrics:

- Per Datacenter
- Aggregated

I have a slight problem with using `without` as a prefix because this means changing the default state would be very confusing. My plan is to pitch that the default should not include per datacenter metrics knowing now how much extra data this creates. We back ourselves into a corner with this name.

Reply:
That makes sense re: naming. I think my confusion stems from using both `aggregate` _and_ `datacenter`. I read that as aggregating **by** datacenter. Maybe `aggregated-only`?

I'm reluctant to change the default behavior as that would be a pretty major breaking change and I know many users rely on the per-datacenter metrics.